docs/source/examples/checks/distribution/trust_score_comparison.ipynb

```diff
@@ -2,106 +2,136 @@
  "cells": [
```

Comment:
<ol><li>Detecting "problematic samples" before labels become available - prediction with low probability are more likely to be wrong.</li></ol><p> </p><p>prediction -&gt; predictions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/644/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>